app/controllers/staff/reports_controller.rb

```diff
@@ -147,12 +146,14 @@ def send_csv
     response.headers["Content-Type"] = "text/csv"
     response.headers["Content-Disposition"] = "attachment; filename=#{ERB::Util.url_encode(@report.description)}.csv"
     response.stream.write ExportActivityToCsv.new(report: @report).headers
-    @projects.each do |project|
-      response.stream.write ExportActivityToCsv.new(activity: project, report: @report).call
-    end
-    @third_party_projects.each do |project|
-      response.stream.write ExportActivityToCsv.new(activity: project, report: @report).call
+    @report_activities.each do |activity|
+      response.stream.write ExportActivityToCsv.new(activity: activity, report: @report).call
     end
     response.stream.close
   end
+
+  def level_c_and_d_activities_for_report(report:)
+    return [] if report.nil?
```

Comment:
Probably makes no different here, but you could also do `return Activity.none if report.nil?` which returns an empty set of activities.

Reply:
Pah! I went looking for that but settled! Thank you!